velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -226,7 +226,9 @@ class HiveConnectorTestBase : public OperatorTestBase {
 class HiveConnectorSplitBuilder {
  public:
   HiveConnectorSplitBuilder(std::string filePath)
-      : filePath_{std::move(filePath)} {}
+      : filePath_{filePath.find("/") == 0 ? "file:" + filePath : filePath} {
```

Comment:
Why we need this prefix to ensure it goes to local fs? IIRC, the default goes to local fs.

Reply:
I don't know either, maybe some legacy workaround, let me remove it to see if it still works